background/test/app.test.js

```diff
@@ -43,38 +43,41 @@ describe('behavior of cloud function', function() {
     } catch (err) {
       console.log("Wasn't able to delete Google Cloud Functions");
     }
-    try {
-      cp.execSync(
-        `firebase firestore:delete --project ${projectId} -r translations`
-      );
-    } catch (err) {
-      console.log("Wasn't able to delete firestore project");
-    }
+    const db = new Firestore({
+      project: projectId,
+    });
+    const res = await db.collection('/translations').get();
+    res.forEach(element => {
+      element.ref.delete();
```

Comment:
is `delete` async?  This looks like it's leaving a promise dangling.  If this *is* async, there are two ways to handle this:

```js
// delete all of the serially, one at a time
for (const element of res) {
  await element.ref.delete();
}

// or do it all at once
await Promise.all(res.map(element => element.ref.delete()));
```